<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiscreteStateWindow</class>
 <widget class="QDialog" name="DiscreteStateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Single/Multi State WIndow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Candidate Matches:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeMatches">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Number of States:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinStates">
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbRealtime">
       <property name="text">
        <string>Realtime</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbLogged">
       <property name="text">
        <string>From logged data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Toggle Frequency (Seconds)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinFreq">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Iterations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinIterations">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblStatus">
         <property name="palette">
          <palette>
           <active>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>120</red>
               <green>120</green>
               <blue>120</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font">
          <font>
           <family>Nirmala UI</family>
           <pointsize>26</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>WAIT</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Min Bits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinMinBits">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Max Bits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinMaxBits">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>IDs to Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listID"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btnAll">
           <property name="text">
            <string>All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNone">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnStart">
     <property name="text">
      <string>Go for it</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeMatches</tabstop>
  <tabstop>spinStates</tabstop>
  <tabstop>rbRealtime</tabstop>
  <tabstop>rbLogged</tabstop>
  <tabstop>spinFreq</tabstop>
  <tabstop>spinMinBits</tabstop>
  <tabstop>spinIterations</tabstop>
  <tabstop>spinMaxBits</tabstop>
  <tabstop>listID</tabstop>
  <tabstop>btnAll</tabstop>
  <tabstop>btnNone</tabstop>
  <tabstop>btnStart</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
